doc/contribute.rst

```diff
@@ -0,0 +1,29 @@
+:orphan:
+
+Contributing to MNE-BIDS
+========================
+
+MNE-BIDS is an open-source software project developed through community effort.
+You are very welcome to participate in the development by reporting bugs,
+writing documentation, or contributing code.
+
+Development takes place on the collaborative platform GitHub at
+`github.com/mne-tools/mne-bids <https://github.com/mne-tools/mne-bids>`_.
+
+.. image:: assets/GitHub.png
```

Comment:
how about adding it to the gh-pages branch directly to avoid bloating master?

Reply:
It's 230 kB, does it really matter?

Reply:
I gave Mainak's comment a thumbs up, but reconsidering it now --> it's probably cleaner to have the documentation source + all assets in one place. So +1 to have it like it is here. The files are small enough for my taste.

Reply:
you can't remove a thumbs up :) I think 230 kB will matter if ever you want to update that image. You won't be able to force push. So if you're certain you'll never change it, then go for it.

Reply:
Maybe the compromise is to include the files in this repo for now, and if we realize during future doc updates that we keep on changing this stuff, we move the images to gh-pages. What do you think?





> you can't remove a thumbs up :)

Yeah there should be an edit history haha :)